gnosis/cowswap
# [Paraswap] migration to v5

# Summary

Fixes #1378

- migration from Paraswap v4 to v5

  # To Test

1. Go to Cowswap
2. Test different swap combinations to check if there is anything unusual in the prices related to the current production version or any other issues related to prices

# Note
- update solver address in the code
